src/helper.cpp

```diff
@@ -151,6 +151,8 @@ bool expandIntervalHint (
     {":month",       {"som",       "eom"}},
     {":quarter",     {"soq",       "eoq"}},
     {":year",        {"soy",       "eoy"}},
+    // Use the "epoch" from the Datetime class, which is 1980-01-01
+    {":all",         {"315532800", "eod"}},
```

Comment:
Time tracking before 1980 and/or in the future is unlikely (and forbidden in the latter case) but I think it is better to expand the `:all` hint  into what it really means, i.e. an open range, instead of introducing artificial boundarys.

Reply:
Defining it as an open range does create a problem with CmdSummary. But perhaps the filter needs to auto close a single day to be closed interval itself and not leave that to CmdSummary?

Reply:
I just modified CmdSummary to only auto close the interval for a day if it was already started.